src/lib/components/customId.svelte

```diff
@@ -27,7 +27,7 @@
             id = null;
         }
 
-        if (!id?.length) {
+        if (id !== null && !id.length) {
```

Comment:
Wouldn't this effect block trigger every time `id` changes so `trackEven()` would be called every time `id` changes?